querydsl-apt/src/main/java/com/querydsl/apt/VisitorConfig.java

```diff
@@ -14,7 +14,7 @@
 package com.querydsl.apt;
 
 /**
- * VisitorConfig defines the entity type specific visiting configuration
+ * {@link VisitorConfig} defines the entity type specific visiting configuration
```

Comment:
This sentence seems a little weird but I couldn't think of a better way to rewrite it.